env.ts

```diff
@@ -0,0 +1,42 @@
+import 'dotenv/config';
+import { z } from 'zod';
+
+const envSchema = z.object({
+    IEXEC_VOUCHER_HUB_ADDRESS: z
+        .string()
+        .regex(/^0x[a-fA-F0-9]{40}$/, 'Invalid Ethereum address')
+        .optional(),
+    IEXEC_VOUCHER_MANAGER_ACCOUNT_INDEX: z
+        .string()
+        .optional()
+        .refine(
+            (val) => val === undefined || /^\d+$/.test(val),
+            'Must be a numeric index if provided',
+        ),
+    IEXEC_VOUCHER_MINTER_ACCOUNT_INDEX: z
+        .string()
+        .optional()
+        .refine(
+            (val) => val === undefined || /^\d+$/.test(val),
+            'Must be a numeric index if provided',
+        ),
+    IS_LOCAL_FORK: z.preprocess(
+        (val) => (typeof val === 'string' ? val.toLowerCase() === 'true' : val === true),
+        z.boolean().default(false),
+    ),
```

Comment:
Can we replace this with `z.boolean()` and use `true/false` instead of `"true"/"false"`?

Reply:
Can you explain ? I'm not sure to understand. You want to do that ?
```suggestion
    IS_LOCAL_FORK: z.boolean().default(false),
,
    ),
```

Reply:
The important fact to keep in mind is that the received values are always strings. If you do that, you will encounter the following issue : 
```
ZodError: [
  {
    "code": "invalid_type",
    "expected": "boolean",
    "received": "string",
    "path": [
      "IS_LOCAL_FORK"
    ],
    "message": "Expected boolean, received string"
  }
]
```

Reply:
Ok I thought `zod` could directly convert `"true"` strings to boolean. All good thanks.